src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -487,6 +542,20 @@ private static IncrementalMethodStubGenerationContext CalculateStubInformation(M
                 // Missing Guid
             }
 
+            // TODO: Error if more than one GeneratedComInterface base interface type.
```

Comment:
Is this still a todo, or does the code below do this?
```suggestion
            // Error if more than one GeneratedComInterface base interface type.
```